CMakeLists.txt

```diff
@@ -32,8 +32,8 @@ if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
     endif ()
 
 elseif (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
-    if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS "16.0.0")
-        MESSAGE(FATAL_ERROR "Clang++ versions older than 16.0 are not supported by QLever")
+    if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS "15.0.0")
+        MESSAGE(FATAL_ERROR "Clang++ versions older than 15.0 are not supported by QLever")
```

Comment:
As long as this compilation doesn't work in fact, I would prefer having an additional
variable to be used here, such that this reads `if (COMPILER_VERSION LESS 16 && !COMPILER_VERSION_CHECK_DEACTIVATED`